packages/scaffold/src/partials/w3m-connecting-header/index.ts

```diff
@@ -1,24 +1,45 @@
-import type { Platform } from '@web3modal/core'
+import { ConnectionController, type Platform } from '@web3modal/core'
 import { LitElement, html } from 'lit'
-import { customElement, property } from 'lit/decorators.js'
+import { customElement, property, state } from 'lit/decorators.js'
 
 @customElement('w3m-connecting-header')
 export class W3mConnectingHeader extends LitElement {
   // -- Members ------------------------------------------- //
   private platformTabs: Platform[] = []
 
+  private unsubscribe: (() => void)[] = []
+
   // -- State & Properties -------------------------------- //
   @property({ type: Array }) public platforms: Platform[] = []
 
   @property() public onSelectPlatfrom?: (platform: Platform) => void = undefined
 
+  @state() private buffering = false
+
+  public constructor() {
+    super()
+    this.unsubscribe.push(
+      ConnectionController.subscribeKey('buffering', val => {
```

Comment:
Let's simplify to `ConnectionController.subscribeKey('buffering', val => (this.buffering = val))` and remove extra function